Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu
```

Comment:
We're migrating from ubi-minimal to ubuntu, I don't see it very optimal....

Reply:
using ubi-minimal as suggested.